src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,136 @@
+package duke.ui;
+
+import java.util.List;
+
+import duke.tasks.Task;
+
+/**
+ * UI of the application
+ */
+public class Ui {
+    /**
+     * Returns success message for setting marked value of a task.
+     *
+     * @param isMark boolean value whether to mark the task
+     * @param task the task to be marked
+     */
+    public String showSuccessMark(boolean isMark, Task task) {
```

Comment:
You may want to rename the function name to something more intuitive, such as `getSuccessMarkMessage`.
`show` might give off the impression that it is a `void` function.

I noticed this in other function names in this class as well.